app/src/main/res/layout/activity_main.xml

```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
     android:layout_width="match_parent"
-    android:layout_height="match_parent" />
+    android:layout_height="match_parent">
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
do we still need this layout? no, right?

Reply:
@luongvo, I don't think we need it anymore. I think let's remove it later. Maybe create a ticket to remove this because there're many files we should remove as well e.g. (nav_graph_main.xml, colors.xml, and so on) What do you think? cc @hoangnguyen92dn

Reply:
@Wadeewee that works 👍